### DISCUSSION TITLE
What's "'ip' is not needed for this connection"?

### DISCUSSION THREAD

**Discussion by HubKing** [neutrinolabs/xrdp#2992] [Q&A] [answered]
Server is Manjaro (Arch derivative) Linux + Gnome. There was an XRDP (AUR) update today, and  after the upgrade, I cannot log in to the computer using Remmina. It keeps showing the following dialogue, and rebooting the computer did not fix it. What is this?

![image](https://github.com/neutrinolabs/xrdp/assets/10900489/b6e5584e-5238-48d4-ae2d-476bbf217bd7)

**Comment by matt335672** [comment, marked as answer]
It looks like the AUR update is a pre-release of v0.10.

There's an `ip=` line in your `xrdp.ini` which can now be removed. It was used to determine how xrdp talks to sesman. This is now done via a UNIX domain socket so there is no need for this parameter.

As for the error itself, have a look in `/var/log/xrdp-sesman.log`.

**Reply by HubKing** [reply]
The log file says the following. The PC is headless, so it does not have any monitor or even a GPU. But until today's update, XRDP was working. I tried to find a way to fix it, but I could not. Could you tell me what I could try?

> [INFO ] Starting X server on display 10: /usr/lib/Xorg :10 -auth .Xauthority -config xrdp/xorg.conf -noreset -nolisten tcp -logfile .xorgxrdp.%s.log 
> [ERROR] waitforx: Unable to open display :10
> [ERROR] X server failed to start
> [INFO ] Session on display 10 has finished.

Installed packages are:
```
aur/xorgxrdp 0.10.0-1 (+13 0.96) (Installed)
    Xorg drivers for xrdp

aur/xrdp 0.10.0_beta.1-1 (+138 0.80) (Installed)
    An open source remote desktop protocol (RDP) server
```

**Reply by matt335672** [reply]
See neutrinolabs/xorgxrdp#294. You may have the same problem.

**Comment by metalefty** [comment]
The screenshot in the first post is not xrdp v0.10. It doesn't have new look of login screen. So I guess the reporter mixed xrdp v0.9 and xorgxrdp v0.10.

**Reply by metalefty** [reply]
xrdp v0.10 should look like this:
![image](https://github.com/neutrinolabs/xrdp/assets/941609/2034b2ea-79b5-4d62-83e8-5a3246c65d4e)

**Reply by matt335672** [reply]
That's interesting - thanks.

I was looking at the messages in the connection log. They're definitely 0.10.x messages. I'll try to work out exactly what's in those Arch packages and see what's going on.
